tests/epyccel/test_arrays.py

```diff
@@ -1476,6 +1476,22 @@ def test_multiple_negative_index_3():
 
     assert np.array_equal(f1(-1, -1, -3), f2(-1, -1, -3))
 
+def test_argument_negative_index_1():
+    a = np.empty(21, dtype=np.int)
+    for i in range(21):
+        a[i] = 1 << i
```

Comment:
You should be able to make this faster by using a list comprehension:
`a = np.array([1 << i for i in range(21)], dtype=np.int)`